lib/gelfx/log_entry.ex

```diff
@@ -185,4 +183,31 @@ defmodule Gelfx.LogEntry do
   def short_message(<<>>, original, _length) do
     original
   end
+
+  defp formatted_full_message({module, func}, level, message, timestamp, metadata) do
+    try do
+      with true <- Code.ensure_loaded?(module),
```

Comment:
These checks could just be removed.
If the apply fails the rescue would catch the exception and run the same `default_formatted_full_massage` call.